Sources/MapboxDirectionsCLI/main.swift

```diff
@@ -2,14 +2,65 @@
 
 import Foundation
 import MapboxDirections
-import SwiftCLI
+import ArgumentParser
 
 
-let match = ProcessCommand< MapMatchingResponse, MatchOptions > (name: "match",
-                                                                 shortDescription: "Command to process Map Matching Data")
-let route = ProcessCommand< MapboxDirections.RouteResponse, RouteOptions>(name: "route",
-                                                                          shortDescription: "Command to process Routing Data")
+struct ProcessingOptions: ParsableArguments {
+    
+    @ArgumentParser.Option(name: [.short, .customLong("input")], help: "Filepath to the input JSON.")
+    var inputPath: String?
+    
+    @ArgumentParser.Option(name: [.short, .customLong("config")], help: "Filepath to the JSON, containing serialized Options data.")
+    var configPath: String?
+    
+    @ArgumentParser.Option(name: [.short, .customLong("output")], help: "[Optional] Output filepath to save the conversion result. If no filepath provided - will output to the shell.")
+    var outputPath: String?
+    
+    @ArgumentParser.Option(name: [.customShort("f"), .customLong("format")], help: "Output format. Supports `text` and `json` formats.")
+    var outputFormat: OutputFormat = .text
+    
+    enum OutputFormat: String, ExpressibleByArgument {
```

Comment:
I would recommend adding `CaseItarable` here to support [these extensions](https://github.com/apple/swift-argument-parser/blob/d2930e8fcf9c33162b9fcc1d522bc975e2d4179b/Sources/ArgumentParser/Parsable%20Types/ExpressibleByArgument.swift#L47-L61)